sdk/route/src/executor/executor.ts

```diff
@@ -0,0 +1,740 @@
+import {
+  AttestedTransferReceipt,
+  Chain,
+  ChainAddress,
+  ChainContext,
+  CompletedTransferReceipt,
+  DestinationQueuedTransferReceipt,
+  Network,
+  RedeemedTransferReceipt,
+  Signer,
+  TokenId,
+  TransactionId,
+  TransferReceipt as _TransferReceipt,
+  TransferState,
+  UniversalAddress,
+  Wormhole,
+  WormholeMessageId,
+  amount,
+  canonicalAddress,
+  deserializeLayout,
+  encoding,
+  finality,
+  guardians,
+  isAttested,
+  isDestinationQueued,
+  isFailed,
+  isRedeemed,
+  isSourceFinalized,
+  isSourceInitiated,
+  nativeTokenId,
+  routes,
+  serializeLayout,
+  signSendWait,
+  toChainId,
+} from "@wormhole-foundation/sdk-connect";
+import "@wormhole-foundation/sdk-definitions-ntt";
+import { NttRoute } from "../types.js";
+import { getReferrerAddress } from "./consts.js";
+import {
+  calculateReferrerFee,
+  fetchCapabilities,
+  fetchSignedQuote,
+  fetchStatus,
+  RelayStatus,
+} from "./utils.js";
+import { Ntt, NttWithExecutor } from "@wormhole-foundation/sdk-definitions-ntt";
+import {
+  isNative,
+  relayInstructionsLayout,
+  signedQuoteLayout,
+} from "@wormhole-foundation/sdk-definitions";
+
+export namespace NttExecutorRoute {
+  export type Config = {
+    ntt: NttRoute.Config;
+    // Referrer Fee in *tenths* of basis points - e.g. 10 = 1 basis point (0.01%)
+    referrerFeeDbps?: bigint;
```

Comment:
IMO it's weird that referrer fee dbps is configurable but not _where_ the fee goes.